Auth/src/jsMain/kotlin/io/github/jan/supabase/auth/setupPlatform.kt

```diff
@@ -23,7 +23,7 @@ actual fun Auth.setupPlatform() {
 
         parseFragmentAndImportSession(afterHash) {
             val newURL = window.location.href.split("?")[0];
```

Comment:
I believe this should also be `split("#")` however I don't have time to change it.

Reply:
Let me check this